crates/rpc/rpc-eth-api/src/core.rs

```diff
@@ -338,6 +339,10 @@ pub trait EthApi<T: RpcObject, B: RpcObject, R: RpcObject, H: RpcObject> {
     #[method(name = "sendRawTransaction")]
     async fn send_raw_transaction(&self, bytes: Bytes) -> RpcResult<B256>;
 
+    /// Sends raw transaction and waits for receipt.
+    #[method(name = "sendRawTransactionSync")]
+    async fn send_raw_transaction_sync(&self, tx: Bytes) -> RpcResult<Option<R>>;
```

Comment:
let's hold off on integrating this into the rpc namespace and limit this to just an api function convenience function instead until we figure out how to expose this

so we can remove this fn here